### ISSUE TITLE
Configure Ruby 2.7 for Heroku PR

### ISSUE THREAD

**Issue by munen** [errbit/errbit#1515]
Would you mind explaining why you closed [this PR](https://github.com/errbit/errbit/pull/1513)? Without this change, there's no way to use the current Heroku stack-20 whilst stack-18 is deprecated. Also, the errbit docs claim to require Ruby 2.7.

I don't mind if you really don't want the change, but closing without comment seems a bit weird. There's also no user connected to the closing action.

**Comment by burnettk** [comment]
i don't think i've ever seen a close event without a user, but i don't use github at my day job, so it might be lack of experience. @stevecrozz , can you see what happened? otherwise, happy to re-open and merge.

**Comment by stevecrozz** [comment]
Very strange. I don't know what happened there, but I reopened it. I'll close this issue in favor of the PR

**Comment by munen** [comment]
Thanks to both of you for the extraordinarily swift and positive response :pray:

**Comment by burnettk** [comment]
it's a lovely day for errbit. i rolled out tip of main to our environments and the projects screen with new configuration suggestions is looking hot. maybe one day errbit will have as much documentation as https://github.com/munen/emacs.d#readme. :)

**Comment by munen** [comment]
*hihi* Thank you for the kind words :pray:

I have used Errbit in various projects for about a decade and am very happy to be able to give a tiny bit back today.

>  maybe one day errbit will have as much documentation as https://github.com/munen/emacs.d#readme. :)

Creating elaborate documentation for Emacs config is a bit easier than other kinds of documentation. Thanks to the built-in [Org mode](https://orgmode.org/), it is quite easy to write literate programs (and config) that double as documentation and are nicely rendered on Github/Gitlab.

From your website, I can see that you're an early Clojure adopter and VIM user. I've been a VIM user for a good decade before finally grasping Lisp through Clojure, as well. And then I got quite interested in Emacs, too.

Thank you for your amazing work on Errbit :bow:
